mkosi/distributions/centos.py

```diff
@@ -36,6 +36,10 @@ def join_mirror(config: MkosiConfig, link: str) -> str:
 
 
 class Installer(DistributionInstaller):
+    @classmethod
+    def pretty_name(cls) -> str:
+        return "CentOS"
```

Comment:
You added this but only implemented it for CentOS? Let's add implementations for the other distributions as well and expose it via `Distribution` so we can use it in more places.